internal/backend/azure/azure.go

```diff
@@ -207,25 +224,39 @@ func (be *Backend) Path() string {
 	return be.prefix
 }
 
+// useAccessTier determines whether to apply the configured access tier to a given file.
+// For archive access tier, only data files are stored using that class; metadata
+// must remain instantly accessible.
+func (be *Backend) useAccessTier(h backend.Handle) bool {
+	notArchiveClass := strings.ToLower(be.cfg.AccessTier) != "archive"
+	isDataFile := h.Type == backend.PackFile && !h.IsMetadata
+	return isDataFile || notArchiveClass
```

Comment:
```suggestion
	isDataFile := h.Type == backend.PackFile && !h.IsMetadata
	return isDataFile || !strings.EqualFold(be.cfg.AccessTier, "archive")
```

Reply:
Let's keep the `notArchiveClass` variable for similarity with `s3.useStorageClass`, but +1 on `strings.EqualFold`.

Reply:
I have accepted @MichaelEischer's recommendation